docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -735,6 +735,22 @@ The output of the query has the following columns:
     - ``array(integer)``
     - The set of field IDs used for equality comparison in equality delete files
 
+``$all_files`` table
+^^^^^^^^^^^^^^^^^^^^
+
+The ``$all_files`` table provides a detailed overview of the data file in all snapshots of the Iceberg table.
```

Comment:
Please keep the line length shorter (~ 80) as done in the rest of the file.

Reply:
updated @findinpath